packages/webb-ui-components/src/components/Avatar/Avatar.tsx

```diff
@@ -32,6 +32,7 @@ export const Avatar: React.FC<AvatarProps> = (props) => {
     darkMode,
     fallback,
     size = 'md',
+    sourceVariant,
```

Comment:
`sourceVariant` = 'uri'

Reply:
It's an optional prop, no need for a default value